source/MaterialXGenShader/GenOptions.h

```diff
@@ -31,6 +31,20 @@ enum ShaderInterfaceType
     SHADER_INTERFACE_REDUCED
 };
 
+
+/// Method to use for directional albedo evaluation
+enum DirectionalAlbedoMethod
```

Comment:
Change this from HW specific option to general options as it's used by both HW and OSL now.